src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_seckey.c

```diff
@@ -139,3 +139,82 @@ uint64_t AppleCryptoNative_SecKeyGetSimpleKeySizeInBytes(SecKeyRef publicKey)
 
     return SecKeyGetBlockSize(publicKey);
 }
+
+#if defined(TARGET_MACCATALYST) || defined(TARGET_IOS) || defined(TARGET_TVOS)
+static CFStringRef GetKeyAlgorithmIdentifier(PAL_KeyAlgorithm keyAlgorithm)
+{
+    if (keyAlgorithm == PAL_KeyAlgorithm_EC)
+        return kSecAttrKeyTypeECSECPrimeRandom;
```

Comment:
It looks like there are a couple variations: kSecAttrKeyTypeECSECPrimeRandom, kSecAttrKeyTypeECDSA, kSecAttrKeyTypeEC.  What's the difference / how was this one chosen?

Reply:
The last two are deprecated and recommend using the first one. 

* https://developer.apple.com/documentation/security/ksecattrkeytypeecdsa
* https://developer.apple.com/documentation/security/ksecattrkeytypeec

Reply:
It's something I adapted from Kevin's original attempt with using EC with the iOS-style data keys. Internally the keys are identical but I guess I should use more specific types here.

Reply:
:facepalm: I was two clicks away from the answers.  If only they still put "(Deprecated)" in the summary description of deprecated things :smile:.